src/dist/replication/lib/throttling_controller.h

```diff
@@ -69,7 +72,7 @@ class throttling_controller
     const std::string &env_value() const { return _env_value; }
 
     // 'delay_ms' is out param when the return type is not PASS.
```

Comment:
'delay_ms' is set when the return type is not PASS.

Reply:
done